dev-tools/mage/dockerbuilder.go

```diff
@@ -141,13 +144,42 @@ func (b *dockerBuilder) dockerSave(tag string) error {
 	// Save the container as artifact
 	outputFile := b.OutputFile
 	if outputFile == "" {
-		outputTar, err := b.Expand(defaultBinaryName + ".docker.tar")
+		outputTar, err := b.Expand(defaultBinaryName + ".docker.gz")
```

Comment:
should it be `tgz` or `tar.gz`?

Reply:
The thing is that package tester looks for packages to test based on the extension, and we already have `tar.gz` for other things, but well, done with more specific regexps.